libraries/AP_InertialNav/AP_InertialNav.h

```diff
@@ -40,13 +40,6 @@ class AP_InertialNav
      */
     virtual nav_filter_status get_filter_status() const = 0;
 
```

Comment:
Changes to libraries should be separated out into their own commit for each library.